### PR TITLE
Add HoS Helmet + Oxygen Tank but it works this time

### PR DESCRIPTION
**Pull request by Geevies**
* Adds a HoS Helmet that uses tactical helmet stats, puts it in HoS Locker
* Adds oxygen tank to HoS locker
* Removes old added helmet from HoS Locker

image
image
image
image
image

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -133,11 +133,13 @@
 		new /obj/item/clothing/suit/armor/hos(src)
 		new /obj/item/clothing/suit/storage/vest/hos(src)
 		new /obj/item/clothing/head/helmet/HoS(src)
+		new /obj/item/clothing/head/helmet/hoshelm(src)
 		new /obj/item/clothing/head/beret/sec/hos(src)
 		new /obj/item/clothing/accessory/badge/hos(src)
 		new /obj/item/clothing/shoes/black_boots(src)
 		new /obj/item/clothing/gloves/black_leather(src)
 		//Tools
+		new /obj/item/clothing/shoes/magboots(src)
```
Comment (Alberyk) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: There are mapped magboots already.

Comment (Geevies) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]:
```suggestion
```

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 2]
```diff
@@ -133,6 +133,7 @@
 		new /obj/item/clothing/suit/armor/hos(src)
 		new /obj/item/clothing/suit/storage/vest/hos(src)
 		new /obj/item/clothing/head/helmet/HoS(src)
```
Comment (Arrow768) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: Old helmet still exists

File: code/modules/clothing/head/helmet.dm [hunk 3]
```diff
@@ -45,6 +45,14 @@
 	armor = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
 	flags_inv = HIDEEARS
 
+/obj/item/clothing/head/helmet/hoshelm
```
Comment (Arrow768) [inline code/modules/clothing/head/helmet.dm]: That path is eh.
Use `/obj/item/clothing/head/helmet/hos` for the (new) helmet and `/obj/item/clothing/head/helmet/hos/cap` for the cap and `/obj/item/clothing/head/helmet/hos/dermal` for the dermal armor patch

File: maps/aurora/aurora-4_mainlevel.dmm [hunk 4]
```diff
@@ -24060,8 +24060,8 @@
 /obj/item/weapon/reagent_containers/spray/pepper,
 /obj/item/clothing/suit/space/void/hos,
 /obj/item/clothing/head/helmet/space/void/hos,
-/obj/item/clothing/head/helmet,
 /obj/item/clothing/shoes/magboots,
+/obj/item/weapon/tank/oxygen,
```
Comment (Karolis2011) [inline maps/aurora/aurora-4_mainlevel.dmm]: Is this intentional?

Comment (Geevies) [reply maps/aurora/aurora-4_mainlevel.dmm]: Yes

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 5]
```diff
@@ -132,7 +132,8 @@
 		new /obj/item/clothing/suit/storage/toggle/armor/hos/jensen(src)
 		new /obj/item/clothing/suit/armor/hos(src)
 		new /obj/item/clothing/suit/storage/vest/hos(src)
-		new /obj/item/clothing/head/helmet/HoS(src)
+		new /obj/item/clothing/head/helmet/hos/cap(src)
+		new /obj/item/clothing/head/helmet/hos/helmet(src)
```
Comment (Geevies) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]:
```suggestion
		new /obj/item/clothing/head/helmet/hos(src)
```

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 6]
```diff
@@ -175,7 +176,8 @@
 		new /obj/item/clothing/under/rank/head_of_security/corp(src)
 		new /obj/item/clothing/suit/storage/vest/hos(src)
 		new /obj/item/clothing/head/beret/sec/hos(src)
-		new /obj/item/clothing/head/helmet/HoS(src)
+		new /obj/item/clothing/head/helmet/hos/cap(src)
+		new /obj/item/clothing/head/helmet/hos/helmet(src)
```
Comment (Geevies) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]:
```suggestion
		new /obj/item/clothing/head/helmet/hos(src)
```

File: code/modules/clothing/head/helmet.dm [hunk 7]
```diff
@@ -38,14 +38,22 @@
...
-/obj/item/clothing/head/helmet/HoS
+/obj/item/clothing/head/helmet/hos/cap
 	name = "head of security hat"
 	desc = "The hat of the Head of Security. For showing the officers who's in charge."
 	icon_state = "hoscap"
 	armor = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
 	flags_inv = HIDEEARS
 
-/obj/item/clothing/head/helmet/HoS/dermal
+/obj/item/clothing/head/helmet/hos/helmet
```
Comment (Geevies) [inline code/modules/clothing/head/helmet.dm]:
```suggestion
/obj/item/clothing/head/helmet/hos
```

File: code/modules/item_worth/worths_list.dm [hunk 8]
```diff
@@ -533,7 +533,8 @@ var/list/worths = list(
 					/obj/item/clothing/head/helmet/tactical = 800,
 					/obj/item/clothing/head/helmet/gladiator = 300,
 					/obj/item/clothing/head/helmet/ert = 1000,
-					/obj/item/clothing/head/helmet/HoS = 800,
+					/obj/item/clothing/head/helmet/hos/helmet = 800,
```
Comment (Geevies) [inline code/modules/item_worth/worths_list.dm]:
```suggestion
					/obj/item/clothing/head/helmet/hos = 800,
```

**Pull request merged**